t/cobrand/brent.t

```diff
@@ -1209,27 +1253,81 @@ FixMyStreet::override_config {
                 $mech->content_contains("How long have you", "Extra question for " . $radio->{type});
                 $mech->back;
             }
+            $mech->back;
         }
 
+        $mech->submit_form_ok({ with_fields => { 'container-choice' => 13 } }, "Choose garden bin");
         $mech->submit_form_ok({ with_fields => { 'request_reason' => 'damaged' } });
         $mech->submit_form_ok({ with_fields => { name => "Test McTest", email => $user1->email } });
         $mech->submit_form_ok({ with_fields => { 'process' => 'summary' } });
         $mech->content_contains('Your container request has been sent');
         my $report = FixMyStreet::DB->resultset("Problem")->order_by('-id')->first;
         is $report->get_extra_field_value('uprn'), 1000000002;
-        is $report->get_extra_field_value('Container_Request_Container_Type'), '6::6';
+        is $report->get_extra_field_value('Container_Request_Container_Type'), '13::13';
         is $report->get_extra_field_value('Container_Request_Action'), '2::1';
         is $report->get_extra_field_value('Container_Request_Reason'), '4::4';
         is $report->get_extra_field_value('Container_Request_Notes'), '';
         is $report->get_extra_field_value('Container_Request_Quantity'), '1::1';
-        is $report->get_extra_field_value('service_id'), '265';
+        is $report->get_extra_field_value('service_id'), '317';
 
         FixMyStreet::Script::Reports::send();
         # No sent email, only logged email
         my $body = $mech->get_text_body_from_email;
         like $body, qr/We aim to deliver this container/;
     };
 
+    subtest 'test requesting a container with payment' => sub {
+        for my $test (
+            { id => 11, name => 'food waste caddy', service_id => 316, pence_cost => 500 },
+            { id => 6, name => 'Recycling bin (blue bin)', service_id => 265, pence_cost => 3000 },
+        ) {
+            subtest "...a $test->{name}" => sub {
+                $mech->get_ok('/waste/12345');
+                $mech->follow_link_ok({url => 'http://brent.fixmystreet.com/waste/12345/request'});
+                $mech->submit_form_ok({ with_fields => { 'container-choice' => $test->{id} } }, "Choose " . $test->{name});
+                $mech->submit_form_ok({ with_fields => { 'request_reason' => 'damaged' } });
+                $mech->submit_form_ok({ with_fields => { name => "Test McTest", email => $user1->email } });
+                $mech->content_contains('Continue to payment');
+                $mech->waste_submit_check({ with_fields => { 'process' => 'summary' } });
+
+                my ( $token, $report, $report_id ) = get_report_from_redirect( $sent_params->{returnUrl} );
+
+                is $sent_params->{items}[0]{amount}, $test->{pence_cost}, 'correct amount used';
+                #check_extra_data_pre_confirm($new_report);
```

Comment:
Is this a to-do for later or does it need to be removed? Or the name of the subroutine being tested?